css/css-multicol/multicol-relative-child-clipping.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<title>Relative child wider than the column container should not be clipped</title>
```

Comment:
Indeed, there should be no clipping here.  In fact, I find it somewhat confusing that the topic of clipping is brought up at all. There's nothing in the test that suggests any clipping at all (i.e. no overflow / clip / etc properties).

Maybe this test could be named something like relative-child-overflowing-container?

Reply:
Fair enough, I think I was confused by the original bug title ("[Multicol] Incorrect clipping when a layer is present between the column and the content layer"). Should be fixed now.